tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "moduleResolution": "nodenext",
```

Comment:
explain me why?

Reply:
commonjs - usually uses require/module.exports but nodenext - import, exports. More so, I needed it in order to import `import { restResources } from '@shopify/shopify-api/rest/admin/2023-07';` from shopify installation